listenbrainz/webserver/static/js/src/stats/UserEntityChart.tsx

```diff
@@ -586,21 +586,25 @@ export default class UserEntityChart extends React.Component<
 document.addEventListener("DOMContentLoaded", () => {
   const domContainer = document.querySelector("#react-container");
   const propsElement = document.getElementById("react-props");
+  const globalPropsElement = document.getElementById("global-react-props");
   let reactProps;
+  let globalReactProps;
   try {
     reactProps = JSON.parse(propsElement!.innerHTML);
+    globalReactProps = JSON.parse(globalPropsElement!.innerHTML);
   } catch (err) {
-    // Show error to the user and ask to reload page
+    // TODO: Show error to the user and ask to reload page
```

Comment:
This block of code is being repeated multiple files, I'll try to extract this into a separate function and reuse it.